docs/platforms/python/profiling/index.mdx

```diff
@@ -75,3 +75,9 @@ sentry_sdk.init(
     },
 )
 ```
+
+<Alert level="warning" title="Deprecation">
+
+Setting profiler options via `_experimental` is deprecated and will be removed in a future release.
```

Comment:
```suggestion
Setting profiler options via `_experiments` is deprecated and will be removed in a future release.
```

Do you mean `_experiments`? That's what I'm seeing in the code sample above this...

I would also move this alert up to line 65 so that users see that it's deprecated before trying to use the code sample.

Reply:
Yes, this should def be `_experiments`! Thanks for pointing this out. Will also move it up.